### ISSUE TITLE
QUESTION: Slow calls, timeouts

### ISSUE THREAD

**Issue by aresares** [GoogleCloudPlatform/opentelemetry-operations-java#104]
Hi after configuring this library in order to send traces on GCP we noticed an increment of the time in REST calls and consequently we get a lot of timeouts from our clients and had to disable and switch again to jaeger . Is this possible? How are send traces to GCP synchronously or asynchronously? We are using the last version of the library

```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>io.opentelemetry</groupId>
                <artifactId>opentelemetry-bom</artifactId>
                <version>1.0.1</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

**Comment by jsuereth** [comment]
TL;DR;By OTel Specification, the processor is 'dumb' and synchronous.

You should make sure you configure a BatchSpanProcessor when using exporters when you *can* run asynchronously (i.e. not cloud-run environment).  We're working on boosting the docs across all the gcp exporters to account for this change.

**Comment by aresares** [comment]
ok thanks it worket now
